src/particles/ImpactXParticleContainer.cpp

```diff
@@ -86,11 +88,9 @@ namespace impactx
         pinned_tile.push_back_real(RealSoA::ux, &(*px.cbegin()), &(*px.cend()));
         pinned_tile.push_back_real(RealSoA::uy, &(*py.cbegin()), &(*py.cend()));
         pinned_tile.push_back_real(RealSoA::pt, &(*pz.cbegin()), &(*pz.cend()));
-
-        //the following should be updated
-        pinned_tile.push_back_real(RealSoA::t, np, 0.0);
-        pinned_tile.push_back_real(RealSoA::q_m, np, 0.0);
-        pinned_tile.push_back_real(RealSoA::w, np, 1.0/np);
+        pinned_tile.push_back_real(RealSoA::q_m, np, qm);
+        amrex::ParticleReal const q_e = 1.60217662e-19;
+        pinned_tile.push_back_real(RealSoA::w, np, bchchg/q_e/np);
 
```

Comment:
It looks like the weight w is now the number of physical particles per macroparticle, as we discussed during the meeting.  Looks ok to me.